pkg/utils/utils.go

```
@@ -146,6 +147,29 @@ func GetCluster(connection *sdk.Connection, key string) (cluster *cmv1.Cluster,
 		return
 	}
 
+	// If we get here we might still be able to get some information from the deleted_clusters information:
+	deletedClustersResponse, err := deletedClustersResource.List().Search(clustersSearch).Size(1).Send()
+	if err != nil {
+		err = fmt.Errorf("Can't retrieve deleted clusters for key '%s': '%v'", key, err)
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)